WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt

```diff
@@ -0,0 +1,398 @@
+decisions = {
+	this_is_sparta = {
+		is_high_prio = yes
+
+		potential = {
+			OR = {
+				culture = tsakonian
+				culture = spartan
+				custom_tooltip = {
+					name = is_greek_and_hellenic
+					hidden_tooltip = {
+						AND = {
+							culture = greek
+							OR = {
+								religion = hellenic_pagan
+								religion = cybelian_pagan
+								religion = neoplatonist
+								religion = mithraic_pagan
+								religion = mastimani
+							}
+						}
+					}
+				}
+			}
```

Comment:
Spartans were technically Greeks. Giving Hellenic Greeks a chance to become Spartan (while giving all pagan Tsakonians the chance) expands the oppurtunities for fun.